client/src/pages/RegisterPage/Register.js

```diff
@@ -1,8 +1,83 @@
-import React from 'react';
+import React,{ useState }  from "react";
```

Comment:
i think you need to add more inputs to be compatible with  data in the db

also you are not managing the inputs and not getting the data or storing it to state